src/controllers/category.js

```
@@ -0,0 +1,34 @@
+const { createBadRequestError, createNotFoundError } = require('~/utils/errorsHelper')
+
+const createCategory = async (req, res) => {
+  const categoryData = req.body
+
+  const newCategory = await categoryService.createCategory(categoryData)
+
+  res.status(201).json(newCategory)
+}
+
+const getCategoryById = async (req, res) => {
+  const id = req.params.id
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)